app/models/identity/responses/SetGuestPasswordResponseCookie.scala

```diff
@@ -0,0 +1,37 @@
+package models.identity.responses
+
+import config.Configuration.GuardianDomain
+import org.joda.time.{DateTime, Seconds}
+import play.api.libs.json.{Json, Reads}
+import play.api.mvc.{Cookie => PlayCookie}
+
+import scala.concurrent.ExecutionContext
+
+// Models a cookie from the cookies field of a successful response from the set guest password endpoint
+// of the identity api
+case class SetGuestPasswordResponseCookie(key: String, value: String, sessionCookie: Option[Boolean] = None) {
+  val isSessionCookie = sessionCookie.getOrElse(false)
+}
+
+// Models the cookies field of a successful response from the set guest password endpoint of the identity api
```

Comment:
nitpick/thought while I'm here:
OK to comment, but my general feeling is that the above comments is the kind of information that should be clear from the code (it's the "what it is" rather than the "what were you thinking" information)
maybe the name is already long, but if we add ...`FromIdentityApi` to the end of the case class, the whole comment can be deleted (and therefore won't go out of date)  However I am happy for whatever tweaks to the field names or type name to make it clear.

Of course naming is a kind of commenting too, and can and does get out of date, but it is less likely, and it can't get separated from its object.